### ISSUE TITLE
inf. values for rpn_encode_boxes

### ISSUE THREAD

**Issue by koosy-koo** [yangxue0827/R2CNN_FPN_Tensorflow#7]
I saw inf. values in the `rpn_encode_boxes` sometimes during training. It causes nan total_error and crashed training. 

I think you need to add `activation_fn=tf.nn.relu` in the rpn_net here: https://github.com/yangxue0827/R2CNN_FPN_Tensorflow/blob/master/libs/rpn/build_rpn.py#L233-L238

I have not tested yet. I will let you know if it fixes.

**Comment by yangxue0827** [comment]
The default activation function is relu：
tf.contrib.slim.conv2d(inputs,
          num_outputs,
          kernel_size,
          stride=1,
          padding='SAME',
          data_format=None,
          rate=1,
          activation_fn=nn.relu,
          normalizer_fn=None,
          normalizer_params=None,
          weights_initializer=initializers.xavier_initializer(),
          weights_regularizer=None,
          biases_initializer=init_ops.zeros_initializer(),
          biases_regularizer=None,
          reuse=None,
          variables_collections=None,
          outputs_collections=None,
          trainable=True,
          scope=None):

**Comment by koosy-koo** [comment]
The issue was gone only with a smaller learning rate (0.0001) for my case. Thanks for your answer.
